File: app/src/main/java/ir/food/operatorAndroid/helper/ContinueProcessing.java
package ir.food.operatorAndroid.helper;

import android.content.Intent;

import ir.food.operatorAndroid.activity.MainActivity;
import ir.food.operatorAndroid.app.MyApplication;

public class ContinueProcessing {

    public static void runMainActivity() {
//        if this not checked, the program will be closed when login fragment open from mainActivity

//            if (MyApplication.currentActivity.toString().contains(MainActivity.TAG)) {
//                FragmentHelper
//                        .taskFragment(MyApplication.currentActivity, LogInFragment.TAG)
//                        .remove();
//                return;
//            }
        MyApplication.handler.postDelayed(() -> {
            MyApplication.currentActivity.startActivity(new Intent(MyApplication.currentActivity, MainActivity.class));
            MyApplication.currentActivity.finish();
        },200);

    }
}

File: app/src/main/java/ir/food/operatorAndroid/app/ContinueProcessing.kt
package ir.food.operatorAndroid.app

import android.content.Intent
import ir.food.operatorAndroid.activity.MainActivity
import ir.food.operatorAndroid.fragment.login.LogInFragment
import ir.food.operatorAndroid.helper.FragmentHelper

class ContinueProcessing {
    fun runMainActivity() {
        //if this not checked, the program will be closed when login fragment open from mainActivity
        if (MyApplication.currentActivity.toString().contains(MainActivity.TAG)) {
            FragmentHelper
                .taskFragment(MyApplication.currentActivity, LogInFragment.TAG)
                .remove()
//            FragmentHelper
//                .taskFragment(MyApplication.currentActivity, CheckVerificationFragment.TAG)
//                .remove()
            return
        }
        if (MyApplication.prefManager.userCode != "0") {
            MyApplication.avaStart()
        }
        MyApplication.handler.post {
            MyApplication.currentActivity.startActivity(
                Intent(
                    MyApplication.currentActivity,
                    MainActivity::class.java
                )
            )
            MyApplication.currentActivity.finish()
        }
    }
}